Spack.md

```diff
@@ -0,0 +1,264 @@
+![Kokkos](https://avatars2.githubusercontent.com/u/10199860?s=200&v=4)
+
+# Kokkos Spack
+
+This gives instructions for using Spack to install Kokkos and developing packages that depend on Kokkos.
+
+## Getting Started
+
+Make sure you have downloaded [Spack](https://github.com/spack/spack) and added it to your path.
+The easiest way to do this is often (depending on your SHELL):
+````bash
+> source spack/share/spack/setup-env.sh
```

Comment:
I think spack will complain if you don't also set SPACK_ROOT in your environment variables

Reply:
Reworded to make this clearer. Thanks for pointing this out.